README.md

```diff
@@ -4,10 +4,64 @@
 [![Coverage Status](https://coveralls.io/repos/github/andela/knights-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/knights-bn-backend?branch=develop)
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
-Barefoot Nomad - Making company travel and accomodation easy and convinient.
-=======
-
+# Barefoot Nomad
 ## Vision
+##### Making company travel and accomodation easy and convinient.
+
+
+## Description
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
----
+### Technonlogies
+- Environment : [Nodejs](https://nodejs.org/)
+- Package Manager : [NPM](https://www.npmjs.com)
+- Compiler : [Babel](https://babeljs.io/)
+- Testing : [Mocha and Chai](https://mochajs.org/)
+- Linting : [ESLint](https://eslint.org/)
+- Object-relational mapping (ORM) : [Sequelize](https://sequelize.org/)
+- Database : [postgresql](https://www.postgresql.org/)
+
+### Requirements and Installation steps
+
+#### You need the following to be able to run the application
+
+[Node](https://nodejs.org/en/download/) a runtime environment for JavaScript
+
+[Postman](https://www.getpostman.com/downloads/) to test the Api endpoints
+
+[Visual studio code](https://code.visualstudio.com/download) for editing and running the app
+
+[PostgreSQL](https://www.postgresql.org/download/) for Database
+
+### Installation
+
+##### A. Clone the project
+1. From your computer, open terminal 
+2. Run `https://github.com/andela/knights-bn-backend` OR [Download](https://github.com/andela/knights-bn-backend/archive/develop.zip) the project
+
+##### B. Setting up the environment
+1. Create a file and name it `.env` in root directory
+2. Find a file named `.env.example`
+3. Copy & paste all environmental variables from `.env.example` to `.env`
```

Comment:
`Copy & paste` may mislead people.
Use something like, "Use .env.example" as a blueprint for your .env file."